### ISSUE TITLE
Build failure: rstudio-server

### ISSUE THREAD

**Issue by lelgenio** [NixOS/nixpkgs#259651]
### Steps To Reproduce

```sh
nix build nixpkgs/d09cd1d7d177a30af5b2efbbc853842b6af9f546#rstudio-server 
```

### Build log

https://gist.githubusercontent.com/lelgenio/193d5bd320325df0f460385b09171b52/raw/dbd1831d299396f6c81ab95b8f93c2b420d2bced/gistfile1.txt

### Notify maintainers

@ciil @cfhammill 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.55, NixOS, 23.05 (Stoat), 23.05.20230924.261abe8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.5`
 - nixpkgs: `/nix/store/dn5wzc9a8hrpml66i6kz5f3q6dr3jjmk-source`
```

**Comment by FliegendeWurst** [comment]
Build works on commit 2e7758c1187c4afc7a90332ff6a396555c276f3b.

**Comment by lelgenio** [comment]
Fixed in 45b1d202eff8fad701bba815cdfc30dea79b91f3
